pyproject.toml

```diff
@@ -34,12 +32,11 @@ dependencies = [
     "litellm>=1.64.0",
     "diskcache>=5.6.0",
     "json-repair>=0.30.0",
-    "tenacity>=8.2.3",
+    "tenacity>=8.2.3", # Needed by litellm to do exponential backoff but not listed in pyproject.toml dependencies (only requirements.txt)
```

Comment:
we don't need this comment, readers without the history context will feel confused